git/git.go

```diff
@@ -37,6 +39,31 @@ func New(directory string) (*Git, error) {
 
 }
 
+func (g *Git) CommitID() (string, error) {
+	cmd := exec.Command(g.GitPath, "rev-parse", "--short", "HEAD")
```

Comment:
This is the API for calling git. The branch management should be done a layer above.